package.json

```diff
@@ -737,6 +743,10 @@
                 {
                     "command": "aws.cdk.help",
                     "when": "!isCloud9"
+                },
+                {
+                    "command": "aws.addSamDebugConfig",
+                    "when": "editorLangId == javascript || editorLangId == python || editorLangId == java || editorLangId == csharp"
```

Comment:
can the `in` clause be used? https://code.visualstudio.com/api/references/when-clause-contexts#in-conditional-operator

Reply:
Unsure, will try tomorrow

Reply:
Should update this to include Go since it's merged now.

Reply:
Yup, merge conflict resolution will include this + other requests.

Reply:
Done, since this is done programmatically I also isCloud9ed it